### PR TITLE
Updated SetCookie to use Set

### PR DESCRIPTION
**Pull request by slobo80**
SetCookie is internal API and therefore it should not be used.

# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

**Comment by GuardRex** [comment]
@slobo80 Change the snippet to `Response.Cookies.Add(MyCookie);`

File: samples/snippets/visualbasic/VS_Snippets_WebNet/Classic HttpResponse.SetCookie Example/VB/source.vb [hunk 1]
```diff
@@ -9,7 +9,7 @@ Public Class Page1: Inherits Page
   Protected Sub Page_Load(sender As Object, e As EventArgs)
 ' <Snippet1>
 MyCookie.Value = CStr(DateTime.Now())
- Response.SetCookie(MyCookie)
+    Response.Cookies.Set(MyCookie)
```
Comment (GuardRex) [inline samples/snippets/visualbasic/VS_Snippets_WebNet/Classic HttpResponse.SetCookie Example/VB/source.vb]: Change to `Response.Cookies.Add(MyCookie)`

**Review by guardrex — DISMISSED** [review]

**Comment by GuardRex** [comment]
I think he'll be fine with it. We'll approve and merge as soon as he gets an :eye: on it. Thanks for your help (and patience) with this.

**Review by rpetrusha — APPROVED** [review]
Thanks, @slobo80. Somehow, I'd overlooked the confusion between Add and Set.

**Pull request merged**